test/error.js

```diff
@@ -13,14 +13,15 @@ describe('error handling', () => {
     assert.throws(humanizingWith({ language: 'bad language' }), Error)
     assert.throws(humanizingWith({ language: '' }), Error)
     assert.throws(humanizingWith({ language: null }), Error)
+    assert.throws(humanizingWith({ language: 'bad language', fallback: null }), Error)
   })
 
   it('throws an error when passed a bad language in a humanizer', () => {
     const humanizer = humanizeDuration.humanizer({
       language: 'bad language'
     })
 
-    function humanizing (options) {
+    function humanizing(options) {
```

Comment:
Same spacing issue here:

```suggestion
    function humanizing (options) {
```

Reply:
Apologies about the spacing issues. VS Code took over.